src/encoded/schemas/library.json

```diff
@@ -282,6 +282,12 @@
             "type": "string",
             "pattern": "(^[0-9]+-[0-9]+$|^[<>][0-9]+$)"
         },
+        "queried_RNP_size_range": {
+            "title": "Queried RNP size range",
+            "description": "The measured size range of queried ribonucleoprotein (RNP) eCLIP control, in kD.",
```

Comment:
please shorten the description to be something like "The measured size range of queried ribonucleoprotein (RNP) in kD." and add "comment" where you can explain that for example this range could be specified in eCLIP experiments.